packages/server/graphql/types/GitLabIntegration.ts

```diff
@@ -144,42 +146,55 @@ const GitLabIntegration = new GraphQLObjectType<any, GQLContext>({
             projectsFullPaths.add(edge?.node?.fullPath)
           }
         })
-        const projectIssues = [] as ProjectIssueConnection[]
-        const errors = [] as Error[]
-        const hasNextPage = true
-
-        const projectsIssuesPromises = Array.from(projectsFullPaths).map((fullPath) =>
-          manager.getProjectIssues({
+        const parsed = args.after && JSON.parse(args.after)
```

Comment:
+1 It would be nice if we could have a `GraphQLJSONType` similar to `GraphQLISO8601Type` so we can verify the type before it reaches the resolver.